jerry-debugger/jerry-client-ws.py

```diff
@@ -729,6 +737,48 @@ def parse_source(debugger, data):
         logging.debug("No pending breakpoints")
 
 
+def src_check_args(debugger, is_src_cmd, args):
```

Comment:
We had a bit of miscommunication, sorry about that. What I meant originally that the try-except block could be extracted to it's own method not the whole thing.